includes/Core/Storage/Data_Encryption.php

```diff
@@ -83,19 +83,26 @@ public function encrypt( $value ) {
 	 * @return string|bool Decrypted value, or false on failure.
 	 */
 	public function decrypt( $raw_value ) {
-		if ( ! extension_loaded( 'openssl' ) ) {
+		if ( ! extension_loaded( 'openssl' ) || ! is_string( $raw_value ) ) {
 			return $raw_value;
 		}
 
-		$raw_value = base64_decode( $raw_value, true );
+		$decoded_value = base64_decode( $raw_value, true );
+
+		if ( false === $decoded_value ) {
+			// phpcs:ignore
+			error_log( "Failed to decode base64 value. Raw value: $raw_value" );
```

Comment:
Let's not log anything here as we don't do this anywhere else and it also probably wouldn't help anyone encountering it since it's a low-level function.